buildspec.yml

```diff
@@ -14,19 +14,11 @@ phases:
       - |
         if [ "$(ls -A $CODEBUILD_SRC_DIR)" ]; then
           cd $CODEBUILD_SRC_DIR
-          # skip hidden folders
-          dirs=$(find . -not -path "\./\.*" -mindepth 1 -maxdepth 1 -type d)
-          echo "Folders to build: $dirs"
-          for directory in $dirs; do
-            cd "$directory"
-            mvn -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn -B clean verify --no-transfer-progress
-            if [ "$?" -ne 0 ] ; then
-                echo "Build failed!"
-                exit 1
-            else
-                cd ".."
-            fi
-          done
+          mvn -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn -B clean install --no-transfer-progress
```

Comment:
Why did we move building per folder?

Reply:
In this commit, I introduced a parent pom.xml and all modules(aws-nimblestudio-studio, etc) are included there. The build from root directory will build all resources so I removed building per folder. Please let me know if you have any concerns. :)